sdk/communication/Azure.Communication.CallAutomation/samples/Sample1_StartDialog.md

```diff
@@ -0,0 +1,46 @@
+# Start Dialog
+
+This sample demonstrates how to add a Dialog Bot to a phone call and start it.
+
+To get started you will need:
+- A Communication Service Resource.  See [README][README] for prerequisites and instructions.
+- A Nuance Mix Bot. See [Nuance Mix][Nuance_Mix] for instructions
+
+## Creating a `CallAutomationClient`
+
+A CallDialog instance requires the creation of a CallAutomation client. This can be authenticated using the connection string acquired from an Azure Communication Resource in the Azure Portal
+```C#
+var connectionString = "<connection_string>"; // Find your Communication Services resource in the Azure portal
+CallAutomationClient callAutomationClient = new CallAutomationClient(connectionString);
+```
+
+## Dialog Dependencies and Creating a `CallDialog` Instance
+
+The Dialog relies on an already running call (see the [Create Call Sample][create_call_sample]) to run. With an existing call, take the CallConnectionId to create the CallDialog instance
+```C#
+var callConnection = client.GetCallConnection(@event.CallConnectionId);
+var callDialog = callConnection.GetCallDialog();
+```
+For an example on managing the connection, refer to the [Quickstart][Quickstart].
+
+## Starting the Dialog
+
+To make a new Dialog, call the `StartDialog` from the `CallDialog` client. Its options parameter requires the input type, the ID of the Bot App, and the DialogContext (this can be an empty one to start).
```

Comment:
>  (this can be an empty one to start)

Replace this with some reference that dialog context depends on the ivr bot type. While this empty context works for now with the specific bot types we use, it's not something that will likely stick

Reply:
Good point. I'll fix the serialization issue with the SDK first and then upload a new version of this sample that has a sample context.